tests/functional_tests/iterator_tests/test_fvm_nabla.py

```diff
@@ -351,6 +352,8 @@ def nabla_sign(n_nodes, out_MXX, out_MYY, pp, S_MXX, S_MYY, vol, node_index, is_
 
 def test_nabla_sign(fencil_processor, lift_mode):
     fencil_processor, validate = fencil_processor
+    if lift_mode != LiftMode.FORCE_INLINE:
```

Comment:
We had a discussion yesterday about the problem of lift and concluded that it is not broken in the part where we thought it is. Let's have a chat next week.

Reply:
We should resolve this separately.